resources/views/layouts/default/flights/simbrief_form.blade.php

```diff
@@ -160,6 +160,7 @@
               <input type="hidden" name="omit_stars" value="0">
               <input type="hidden" name="cruise" value="CI">
               <input type="hidden" name="civalue" value="AUTO">
+              <input type="hidden" name="static_id" value="{{ $user->ident.'_'.$flight->id }}">
```

Comment:
Have this static ID generated in the controller. It's easier to find.

Reply:
Done